fwupd/metainfo.xml.in

```diff
@@ -2,8 +2,8 @@
 <!-- Copyright 2020 Evan Lojewski <github@meklort.com> -->
 <component type="firmware">
     <id>@APP_STREAM_PREFIX@.@TARGET_ID@</id>
-    <name>BCM5719</name>
-    <name_variant_suffix>@TARGET_NAME@</name_variant_suffix>
+    <name>@TARGET_NAME@</name>
+    <name_variant_suffix>BCM5719</name_variant_suffix>
     <summary>Firmware for the @TARGET_NAME@ BCM5719 NIC</summary>
```

Comment:
Can you also change the categories section to be:

  <categories>
    <category>X-NetworkInterface</category>
    <category>X-Device</category>
  </categories>

then we get a better icon, and still fall back to the device icon for older clients.